### PR TITLE
Filter packages in Jenkins Powershell

### PR DESCRIPTION
**Pull request by andrewkroh**
This filters the packages used to create the beatname.test binary used in system tests. We needed to filter the tools in scripts/cmd because the code should not be included in the test binary.

### CODE REVIEW

**Comment by andrewkroh** [comment]
jenkins, test it (the generator test had a weird failure)

**Comment by ph** [comment]
jenkins test this please

File: dev-tools/jenkins_ci.sh [hunk 1]
```diff
@@ -19,12 +19,18 @@ jenkins_setup
...
   echo "Running cleanup..."
   rm -rf $TEMP_PYTHON_ENV
-  make stop-environment || true
-  make fix-permissions || true
-  echo "Killing all running containers..."
-  docker ps -q | xargs -r docker kill || true
-  echo "Cleaning stopped docker containers and dangling images/networks/volumes..."
-  docker system prune -f || true
+
+  if docker info > /dev/null ; then
```
Comment (ruflin) [inline dev-tools/jenkins_ci.sh]: How is this change related?

Comment (andrewkroh) [reply dev-tools/jenkins_ci.sh]: It’s a separate problem. (So do a rebase and merge to keep the commits separate.) This relates to the infra issue linked above (Mac provisioning).

Comment (ruflin) [reply dev-tools/jenkins_ci.sh]: Got it, didn't see that one.
